pkg/label/label.go

```diff
@@ -7,8 +7,10 @@ const (
 
 	ManagedBy = "giantswarm.io/managed-by"
 
-	// Labels, used in legacy cluster namespaces
+	// LegacyCustomer Labels, used in legacy cluster namespaces
 	LegacyCustomer = "customer"
+
+	NOTES = "giantswarm.io/notes"
```

Comment:
Coud we use https://github.com/giantswarm/k8smetadata/blob/dff3b2358977e13c90479c66e21c728a96ddfe6d/pkg/annotation/general.go#L12 here?

Reply:
Cool, did not know about them, thanks! :)